workshops/personal_website/README.md

```diff
@@ -0,0 +1,575 @@
+the HTML.
+
+To **link** the CSS file:
+
+1. Type the code for the **link tag** (written below), into the **`head`**
+   section of the code.
+
+   ```html
+   <link rel="stylesheet" href="styles.css">
+   ```
+
+> ![](img/css_add_link_tag.gif)
+
+> Notes:
+>
+> - **`link`** is the tag name
+> - **`link`** tags always belong in the **`head`** of the HTML document
+> - **`link`** is a **self closing tag** like **`img`**
+> - **`href`** is an  **attribute** of the link tag and specifies the location
+>   of the CSS file. It stands for **hypertext reference**.
+> - **`rel`** is an **attribute** that tells the browser to interpret the
+    linked file as a **stylesheet**
+
+### 3) Using CSS to Resize the Image
+
+Now that we linked our CSS file to our HTML file, let's write some CSS to make
+the image smaller:
+
+1. Double click **`styles.css`** to open it
+2. Then add the below CSS code to make the image smaller by typing the
+   following code into the CSS file.
+
+    ```css
+    img {
+        width: 200px;
+    }
+    ```
+3. Then to save your changes, click **"File"** → **"Save"** (or use the shortcut
+   **CTRL + S / Command + S**)
+
+Yay! Our image got smaller!
+
+> ![](img/css_img_width.gif)
+
+**Understanding The CSS More Deeply**
+
+If we were to translate the code into English, this
+
+```css
+img {
+    width: 200px;
+}
+```
+
+ would say
+
+- **Select** every **`img` tag** on this HTML page and set the **width** to be
+  **`200px`**
+
+> **`px`** stands for **pixels**. **Pixels** are the tiny dots that make up a
+> computer screen. By setting the width to **200px**, we are saying we want
+> every image on the page to span the length of 200 pixels (which is about 2
+> inches on a screen). If you want to learn more about pixels, you can Google
+> ["What is a pixel?"](https://www.google.com/#q=what+is+a+pixel)
+
+### 4) Centering the Body
+
+Then to center everything on the page
+
+1. Type the below CSS code underneath our previous CSS code. _Remember, don't
+   copy and paste._
+
+   ```css
+   body {
+       text-align: center;
+   }
+   ```
+3. Save the file by clicking **"File"** → **"Save"** (or use the shortcut
+   **CTRL + S / Command + S**)
+
+> ![](img/css_text_align_center.gif)
+
+Now everything should be centered on the page.
+
+#### Understanding More Deeply
+
+If we were to translate the code into English, this
+
+```css
+body {
+    text-align: center;
+}
+```
+
+would  say:
+
+- **Select** every **`body` tag** on this HTML page and set the **alignment** of
+  the **text** within the body tag to be **centered**
+
+This centers everything on our page because all of the content in our HTML file
+is written inside the body tag.
+
+And because everything is in the **body** tag, everything is centered.
+
+> You can Google ["CSS text align"]
+  (https://www.google.com/?gws_rd=ssl#q=css+text+align) to see what other
+  alignment options are available besides **`center`**
```

Comment:
Same thing as previous blockquotes. Needs to have `>` at the beginning of each line.